src/Validator/Rules/FieldsOnCorrectType.php

```diff
@@ -84,6 +84,10 @@ private function getSuggestedTypeNames(Schema $schema, $type, $fieldName)
             $interfaceUsageCount  = [];
 
             foreach ($schema->getPossibleTypes($type) as $possibleType) {
+                if (! $possibleType instanceof ObjectType) {
+                    continue;
+                }
```

Comment:
`$schema->getPossibleTypes()` is supposed to always return an array of `ObjectType`, so this seems redundant.

Reply:
Yes, thank you. Removed in 801033f